lib/gateway/Shard.js

```diff
@@ -688,6 +689,7 @@ class Shard extends EventEmitter {
                         * @prop {String} oldUser.discriminator The discriminator of the user
                         * @prop {String?} oldUser.avatar The hash of the user's avatar, or null if no avatar
                         * @prop {String?} oldUser.globalName The globally visible display name of the user
+                        * @prop {String?} oldUser.avatarDecoration The hash of the user's avatar decoration, or null if no avatar decoration
```

Comment:
`avatarDecoration` -> `avatarDecorationData`
```suggestion
                        * @prop {Object?} oldUser.avatarDecorationData The data of the user's avatar decoration, or null if there is no avatar decoration
```